### ISSUE TITLE
htmlwidgets _posts/libs not loading in jekyll rendered site

### ISSUE THREAD

**Issue by bbest** [data-lessons/NEON-R-Make-Pretty-Maps-Plots#4]
![image](https://cloud.githubusercontent.com/assets/2837257/10768678/94d59dd6-7c9f-11e5-9160-58f0f472a3f1.png)

See [README.md](https://github.com/data-lessons/NEON-R-Make-Pretty-Maps-Plots/blob/gh-pages/README.md) for attempts so far in getting to work.

**Comment by lwasser** [comment]
Got it. Thanks Ben  for posting this!
I will be chatting with a few folks about jekyll and building these lessons  in December! In the meantime, as we are working these out, we can post the Rstudio rendered html files. Is that ok with you for the time being?

Leah
--
Leah A. Wasser, Ph.D.
Supervising Scientist
Education / Data Products
National Ecological Observatory Network (NEON)
Boulder, Colorado
--------------------
For free tutorials and resources on working with spatio-temporal data:
http://www.neondataskills.org<http://www.neondataskills.org/>

**Comment by bbest** [comment]
Yep, sounds good. Just checked in a simplified `.nojekyll` version which uses the knitted Rmarkdown outputs and brews a listing into the index.html. See http://data-lessons.github.io/NEON-R-Make-Pretty-Maps-Plots.

**Comment by bbest** [comment]
Worth noting that prettier sites using htmlwidgets are done here:

* [rstudio/leaflet at gh-pages](https://github.com/rstudio/leaflet/tree/gh-pages)
* [rstudio/DT at gh-pages](https://github.com/rstudio/DT/tree/gh-pages)
* [rstudio/dygraphs at gh-pages](https://github.com/rstudio/dygraphs/tree/gh-pages)

as mentioned in [Using htmlwidgets graphics with servr::jekyll() · Issue #8 · yihui/knitr-jekyll](https://github.com/yihui/knitr-jekyll/issues/8).

**Comment by lwasser** [comment]
cool -- thank you. Should we keep this open so it remains a "to do" ? we will figure it out! links are greatly appreciated!

**Comment by bbest** [comment]
Sure, as you wish -- reopened. I previously closed it because I committed a solution to render and display posts (see the [README.md](https://github.com/data-lessons/NEON-R-Make-Pretty-Maps-Plots/blob/gh-pages/README.md)). I like the ability to knit a standalone post on my desktop and see it's fully rendered output. If I change a post then I just run make.R to update the index.html. Jekyll doesn't play nicely with htmlwidgets, hence `.nojekyll`.

You might consider opening a separate issue that tries to reconcile bringing these lessons together into a single site, factoring in this closed issue as a factor for consideration (if you want to include htmlwidgets) and one possibility to generate output (ie by skipping use of Jekyll and opting for simpler rendering methods like brew and Makefile).

**Comment by lwasser** [comment]
great suggestion! i'll do that now. and i'll make it a point to bring it up in our discussion in December. i am meeting with someone who is a Make file pro. i am sure he can help with these issues! They will run across all repos! Thanks ben :+1:
